blockonomics-woocommerce.php

```diff
@@ -531,10 +530,10 @@ function blockonomics_update_db_check() {
 
     $installed_ver = get_site_option( 'blockonomics_db_version' );
     if ( $installed_ver != $blockonomics_db_version ) {
-        // Example function to demonstrate table changes between upgrade versions
-        // if ($installed_ver < 1.0) {
-        //     $wpdb->query("ALTER TABLE $table_name DROP transaction;");
-        // }
+        $table_name = $wpdb->prefix . 'blockonomics_orders';
+        if ($installed_ver < 1.1) {
+            $wpdb->query("ALTER TABLE $table_name DROP time_remaining;");
```

Comment:
As far I remember dbDelta will automatically drop the column when `blockonomics_create_table` is called so this is uncessary. Please verify with real table having values

Reply:
It seems like dbDelta cannot delete columns. ([source](https://wordpress.stackexchange.com/a/76930))